src/ape_ethereum/ecosystem.py

```diff
@@ -467,10 +467,14 @@ def decode_calldata(self, abi: Union[ConstructorABI, MethodABI], calldata: bytes
     def decode_returndata(self, abi: MethodABI, raw_data: bytes) -> Tuple[Any, ...]:
         output_types_str_ls = [o.canonical_type for o in abi.outputs]
 
-        try:
-            vm_return_values = decode(output_types_str_ls, raw_data)
-        except (InsufficientDataBytes, NonEmptyPaddingBytes) as err:
-            raise DecodingError(str(err)) from err
+        if raw_data:
+            try:
+                vm_return_values = decode(output_types_str_ls, raw_data, strict=False)
```

Comment:
second change: use `strict=False` here to see if that resolves the problem. I remember having to do this somewhere else because of node oddities (probably hardhat!)